code/modules/antagonists/heretic/magic/wave_of_desperation.dm

```diff
@@ -11,7 +11,7 @@
 	school = SCHOOL_FORBIDDEN
 	cooldown_time = 5 MINUTES
 
-	invocation = "F'K 'FF."
+	invocation = "Kher' Sekh-em waaef'k"
```

Comment:
Given this spell is called "Wave of Desperation" I think it deserves an exclamation mark

Reply:
the og didnt.....